.github/workflows/deploy-docs-site.yml

```diff
@@ -13,25 +13,39 @@ on:
 jobs:
   dummy-job:
     name: Deploy Docs Assets
+    if: github.event_name == 'push' || github.event_name == 'workflow_dispatch'
     runs-on: ubuntu-latest
     steps:
       - name: Check out the repository
-        if: github.event_name == 'push' || github.event_name == 'workflow_dispatch'
         uses: actions/checkout@v3
         with:
           fetch-depth: 0
 
       # Node.js is needed for Yarn
       - name: Setup Yarn
-        if: github.event_name == 'push' || github.event_name == 'workflow_dispatch'
         uses: actions/setup-node@v2
         with:
           node-version: '16.14.0'
           cache: 'yarn'
           cache-dependency-path: docusaurus
 
       - name: Run Docusaurus
-        if: github.event_name == 'push' || github.event_name == 'workflow_dispatch'
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         run: ./tools/bin/deploy_docusaurus
+
+      - name: Notify Slack channel on failure
+        uses: abinoda/slack-action@master
```

Comment:
Would you mind switching to the official slackapi action?
https://github.com/slackapi/slack-github-action

Reply:
Already merged, nevermind =)

Reply:
There's more workflows to switch to the official API, which means we might want to start a separate issue for OSS repo. Or both repos.